pkg/compiler/codegen.go

```diff
@@ -834,6 +858,32 @@ func (c *codegen) Visit(node ast.Node) ast.Visitor {
 	return c
 }
 
+func (c *codegen) pushLabel(name string, size int) {
```

Comment:
Can we name these in some other fashion? We have `newLabel`, `setLabel`, `generateLabel`, `getLabelOffset` already and if you're to add `pushLabel`, `dropLabel` that work with a different structure and have some different semantics, I think it'll be a mess. Something around branching and stack?

Reply:
`(push/drop)StackLabel` ?